examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -669,6 +669,16 @@ extern "C" void app_main()
                    [=]() {
                        ESP_LOGI(TAG, "Opening QR code screen");
                        ESP_LOGI(TAG, "QR CODE Text: '%s'", qrCodeText.c_str());
+                       uint16_t discriminator;
+                       if (ConfigurationMgr().GetSetupDiscriminator(discriminator) == CHIP_NO_ERROR)
+                       {
+                           ESP_LOGI(TAG, "Setup discriminator: %u (0x%x)", discriminator, discriminator);
```

Comment:
Hi @cecille This is helpful. Just a suggestion: Can you add the print statements in the function `createSetupPayload` where the discriminator and the setup pin code are already populated.

Reply:
Sure - I'll add them there too. I want them on the QR code screen as well so I don't have to go back in the logs when I'm looking for them.